src/App.tsx

```diff
@@ -1,98 +1,113 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
Do not forget commit and puch package.json as well to avoid not working app in case other developer use your code
![2024-02-06_11h45_44](https://github.com/mate-academy/react_autocomplete/assets/61904995/8cc7328a-358d-49f0-a945-558b784a0664)